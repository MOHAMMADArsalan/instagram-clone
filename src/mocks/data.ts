export const imageUrl = 'https://notjustdev-dummy.s3.us-east-2.amazonaws.com/images/1.jpg ';