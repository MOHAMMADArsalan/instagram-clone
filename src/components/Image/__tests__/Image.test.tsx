import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Image } from '../';
import { imageUrl } from '../../../mocks/data';


describe('Image', () => {
    let renderImage: any;
    beforeEach(() => {
        renderImage = renderer.create(<Image source={{ uri: imageUrl }} />);
    })

    it('should match the snapshots', () => {
        expect(renderImage.toJSON()).toMatchSnapshot();
    })

    it('should has image source', () => {
        expect(renderImage.root.props.source).toEqual({ uri: imageUrl })
    })
})