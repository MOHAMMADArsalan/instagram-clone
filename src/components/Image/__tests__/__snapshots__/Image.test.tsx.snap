// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should match the snapshots 1`] = `
<Image
  source={
    Object {
      "uri": "https://notjustdev-dummy.s3.us-east-2.amazonaws.com/images/1.jpg ",
    }
  }
/>
`;
